sdk/storage/azure-storage-file/azure/storage/file/file_client.py

```diff
@@ -339,6 +358,28 @@ def create_file( # type: ignore
         :type metadata: dict(str, str)
         :param int timeout:
             The timeout parameter is expressed in seconds.
+        :param file_attributes:
+            The file system attributes for files and directories.
+            If not set, indicates preservation of existing values.
+            Here is an example for when the var type is str: 'Temporary|Archive'
```

Comment:
This comment contradicts the default value you have set in the method signature:
```
file_attributes="none"
```